src/Features/Core/Portable/AddParameter/RegisterFixData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.AddParameter
+{
+    internal class RegisterFixData<TArgumentSyntax> where
```

Comment:
often we use a name like AnalysisResult, or something to that effect.  But it's not a big deal.

Reply:
We also do have a `CodeFixData` type in the same folder. That's why I kept the name.